public/application.tsx

```diff
@@ -1,21 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { HashRouter as Router } from 'react-router-dom';
 import { AppMountParameters, CoreStart } from '../../../src/core/public';
-import { AppPluginStartDependencies } from './types';
 import { QueryInsightsDashboardsApp } from './components/app';
 
-export const renderApp = (
-  { notifications, http }: CoreStart,
-  { navigation }: AppPluginStartDependencies,
-  { appBasePath, element }: AppMountParameters
-) => {
+export const renderApp = ({ chrome }: CoreStart, { element }: AppMountParameters) => {
+  chrome.setBreadcrumbs([{ text: 'Query insights' }]);
```

Comment:
why do we need this? Is this specific to chrome?

Reply:
this is just how breadcrumbs are set for dashboard plugins